debian/ndnsec-tools.postinst

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+set -e
+
+if ! ndnsec-get-default >/dev/null
+then # generate keys for this machine
+  ndnsec-keygen /localhost/operator | ndnsec-install-cert -
+fi
+
+# support EOS
+if [ -f /etc/lsb-release]; then
+  . /etc/lsb-release
+  [ "x${DISTRIB_ID}" = "xEndless" ] && mv /etc/ndn /usr/ndn/etc && ln -sf /usr/ndn/etc /etc/ndn
```

Comment:
Swap with `if` instead of `&&`s as the line is borderline unreadable. In general, we strongly discourage using chained logic operators (`||`/`&&`) on a single line.